### ISSUE TITLE
Error: Can't find variable: customVersionComparator

### ISSUE THREAD

**Issue by Boubaker93** [SudoPlz/sp-react-native-in-app-updates#2]
Hello;

I'm trying to test this library it works fine when i put a versionCode >= to the one in play store. it gives me a response like this : 

```
{
    "other": {
        "isFlexibleUpdateAllowed": false,
        "isImmediateUpdateAllowed": false,
        "packageName": "com.test.test",
        "totalBytes": 0,
        "updateAvailability": 1,
        "versionCode": 3
    },
    "reason": "status: 1 means there's no new version available",
    "shouldUpdate": false
}
```

But when i but my versionCode < to the one in play store to simulate an outdated version i get this error : 
``[Error: sp-react-native-in-app-updates checkNeedsUpdate error: ReferenceError: Can't find variable: customVersionComparator]``

even though in the README the customVersionComparator is not required. 
Am i doing something wrong ? here is how i'm using the lib : 

```
const inAppUpdates = new SpInAppUpdates();

    inAppUpdates
      .checkNeedsUpdate({
        curVersion: '3.0.3',
      })
      .then(result => {
        console.log(result);
        if (result.shouldUpdate) {
          inAppUpdates.startUpdate({
            updateType: SpInAppUpdates.UPDATE_TYPE.FLEXIBLE, // android only, on iOS the user will be promped to go to your app store page
          });
        }
      })
      .catch(error => {
        console.log(error);
      });
```

And i'm using React-Native Version 0.62.2

**Comment by Gustash** [comment]
#4 fixes this issue

**Comment by SudoPlz** [comment]
Fixed on 0.1.0
